### ISSUE TITLE
[BUG] Wrong import from statsforecast when installing u8darts-torch

### ISSUE THREAD

**Issue by JeroenPeterBos** [unit8co/darts#1197]
**Describe the bug**
There are import errors when I try to import `TFTModel` after installing using `u8darts-torch` using conda.

**To Reproduce**
I run the following commands from an M1 macos device.

```
conda create -n bug-example -c conda-forge pytorch u8darts-torch -y
conda activate bug-example
python
>>> from darts.models import TFTModel
```

Which returns:
```
from statsforecast.models import croston_classic, croston_optimized, croston_sba
ImportError: cannot import name 'croston_classic' from 'statsforecast.models'
```

I checked `statsforecast` and indeed `croston_classic` does not exist, only `_croston_classic` and `CrostonClassic`.

**Expected behavior**

**System (please complete the following information):**
 - Python version: 3.8
 - darts version: 0.20.0

**Additional context**
I am running this from an M1 mac with OS 12.5.1
_Additionally, I first tried to install `u8darts-all` using `conda create -n test python=3.8 pytorch u8darts-all`, but that could not find any satisfable dependency configuration._

**Comment by JeroenPeterBos** [comment]
After some further digging I found that this issue is fixed in this PR that has already been merged: [](https://github.com/unit8co/darts/pull/1171)

If anyone else runs into this issue, you can probably solve it by explicitly installing `statsforecasts==0.6.0`.

**Comment by hrzn** [comment]
Yes, this should be fixed in 0.21.0 with `statsforecasts>0.6.0` as well.
